### DISCUSSION TITLE
2.1.0 beta for Forge 1.17

### DISCUSSION THREAD

**Discussion by Draylar** [Draylar/identity#326] [General]
Please provide me with your feedback while using the 2.1.0 beta version for Forge 1.17!

<hr /><em>This discussion was created from the release <a href='https://github.com/Draylar/identity/releases/tag/2.1.0-beta-1.17-forge'>2.1.0 beta for Forge 1.17</a>.</em>

**Comment by Ashtar084** [comment]
please can make for 1.18.1 pls pls pls :D
